apps/google-analytics-4/frontend/src/components/config-screen/GoogleAnalyticsConfigPage.tsx

```diff
@@ -169,8 +174,9 @@ export default function GoogleAnalyticsConfigPage() {
           isInEditMode={isInEditMode}
           onInEditModeChange={handleInEditModeChange}
           onIsValidServiceAccount={handleIsValidServiceAccount}
+          onHasServiceCheckErrorsChange={handleHasServiceCheckErrorsChange}
         />
-        {isAppInstalled && !isInEditMode && (
+        {isAppInstalled && !isInEditMode && !hasServiceCheckErrors && (
```

Comment:
From what I remember from our conversations the decision was to not let the user even proceed past the service account key step until everything has passed.

However, there is an edge case we kicked down the road for when everything was installed correctly with the properties and content types being setup. But then later the user updates the service key incorrectly for example. What should we do then since we'll have existing properties and content types? Right now it won't show them, but there's a good argument to do show them as well along witht eh errors.